packages/jazz-browser/src/createBrowserContext.ts

```diff
@@ -139,7 +139,7 @@ export async function createJazzBrowserGuestContext(
       context.done();
     },
     logOut: () => {
-      context.logOut();
+      return context.logOut();
```

Comment:
The tests showed up that the logOut wasn't waiting for the credentials to be deleted...